deploy/docker/start.sh

```diff
@@ -37,8 +39,19 @@ tail -f /opt/emqx/log/erlang.log.1 &
 #          you must let user know emqx crashed and stop this container,
 #          and docker dispatching system can known and restart this container.
 IDLE_TIME=0
+ERTS="/opt/emqx/erts-$ERTS_VSN"
 MGMT_CONF='/opt/emqx/etc/plugins/emqx_management.conf'
-MGMT_PORT=$(sed -n -r '/^management.listener.http.port[ \t]=[ \t].*$/p' $MGMT_CONF | sed -r 's/^management.listener.http.port = (.*)$/\1/g')
+MGMT_SCHEMA="$(find /opt/emqx/lib -name "emqx_management*" -type d)/priv/emqx_management.schema"
+
+# prevent interpretation by cuttlefish
+_EMQX_NODE_NAME="$EMQX_NODE_NAME"
+unset EMQX_NODE_NAME
+
+MGMT_CONFIG_ARGS=`$ERTS/bin/escript /opt/emqx/bin/cuttlefish -i $MGMT_SCHEMA -c $MGMT_CONF -d /tmp/configs generate`
```

Comment:
we need to extend cuttlefish command with a field value query option.
it does not have to be like the `jq` command, but a single value print would be enough.
e.g.
`$ERTS/bin/escript /opt/emqx/bin/cuttlefish -i $MGMT_SCHEMA -c $MGMT_CONF --print-field 'management.listener.http.port'`